server/data/facility-promos.js

```diff
@@ -105,7 +105,7 @@ export const dailyTourPromo: EventPromo = {
   start: null,
   end: null,
   description: null,
-  dateString: 'Every day',
+  dateString: 'Tuesday-Sunday',
```

Comment:
Should this be an en-dash with spaces either side? I think that's the house style for dates.